indico/modules/events/editing/templates/management/filetypes.html

```diff
@@ -1,9 +1,19 @@
 {% extends 'events/editing/management/base.html' %}
```

Comment:
What about creating an `events/editing/management/type_base.html` template which sets the title and back url, and then inherit from it here?

Reply:
How many templates will inherit form this, though? `filetypes.html` and `editable_type.html` don't have the same titles/subtitles.

Reply:
Yeah those two for now. Won't the titles and back url be the same? Only subtitle and content are going to be different..

Reply:
Ah sorry, misread the names. filetypes and reviewable conditions will inherit from it.

Reply:
I didn't think of reviewable conditions. If @Leats 's PR is ready soon I can wait for it to be merged and rebase on top of it while adding this new template as well.